gandalf.js

```diff
@@ -1,4 +1,4 @@
-const mergedArrays = [...array1, ...array2];
```

Comment:
SCRIPT-This new feature lacks unit tests. Could you add some to ensure everything works correctly?